src/Esfa.Vacancy.Register.Application/Queries/SearchApprenticeshipVacancies/SearchApprenticeshipVacanciesQueryHandler.cs

```diff
@@ -33,39 +28,10 @@ public class SearchApprenticeshipVacanciesQueryHandler : IAsyncRequestHandler<Se
             if (!validationResult.IsValid)
                 throw new ValidationException(validationResult.Errors);
 
-            var searchParameters = new VacancySearchParameters
-            {
-                StandardSectorCodes = await ValidateStandardCodes(request.StandardCodes.Select(int.Parse))
-            };
+            var searchParameters = await _vacancySearchParametersConverter.ConvertFrom(request);
```

Comment:
I like that you've moved out that code into a separate class, makes testing each in isolation much easier, for the win!